### PR TITLE
[admin] merging fixes

### PR DESCRIPTION
**Pull request by cgiger00**
messed up some things in the merge i guess, mostly teleop related @CameronTressler

### CODE REVIEW

**Comment by cgiger00** [comment]
too many problems will fix later

File: base_station/gui/src/components/AutonTask.vue
Comment (CameronTressler) [inline base_station/gui/src/components/AutonTask.vue:30]: @jnnanni I had to add these breaklines to avoid overlapping text for some reason. Not really sure what happened here, at some point this week can you look at it? We shouldn't need to use breaklines to do this. (Not necessary to change before merging this PR I don't think)

**Review by CameronTressler — APPROVED** [review]

**Comment by CameronTressler** [comment]
Ready to merge when you are @cgiger00

**Pull request merged**
